### PR TITLE
fix: fix caption button fluent theme and default theme.

### PR DESCRIPTION
**Pull request by rabbitism**
## What does the pull request do?

Fix CaptionButtons fluent/simple theme.

## What is the current behavior?

Type and name in selector cannot match any elements.

## Checklist

- [ ] Added unit tests (if possible)?
- [ ] Added XML documentation to any related classes?
- [ ] Consider submitting a PR to https://github.com/AvaloniaUI/Documentation with user documentation

### CODE REVIEW

**Comment by robloo** [comment]
Hm, this must have been broken a while. I dont think it's my fault and never even updated this control's template. Anyway, thanks for finding this one too!

File: src/Avalonia.Themes.Fluent/Controls/CaptionButtons.xaml [hunk 1]
```diff
@@ -41,7 +41,7 @@
                   Theme="{StaticResource FluentCaptionButton}"
                   IsVisible="False">
             <Viewbox Width="11" Margin="2">
-              <Path Name="FullScreenButtonPath"
+              <Path Name="PART_FullScreenButtonPath"
```
Comment (robloo) [inline src/Avalonia.Themes.Fluent/Controls/CaptionButtons.xaml:44]: This isn't actually a template part which is defined to be a control used in code behind.

Comment (rabbitism) [reply src/Avalonia.Themes.Fluent/Controls/CaptionButtons.xaml:44]: Ohh, that makes sense. I will submit another commit to revert it.

File: src/Avalonia.Themes.Fluent/Controls/CaptionButtons.xaml [hunk 2]
```diff
@@ -61,7 +61,7 @@
               <Viewbox.RenderTransform>
                 <RotateTransform Angle="-90" />
               </Viewbox.RenderTransform>
-              <Path Name="RestoreButtonPath"
+              <Path Name="PART_RestoreButtonPath"
```
Comment (robloo) [inline src/Avalonia.Themes.Fluent/Controls/CaptionButtons.xaml:64]: Same as above, I don't think this should have the PART_ prefix.

File: src/Avalonia.Themes.Fluent/Controls/CaptionButtons.xaml [hunk 3]
```diff
@@ -80,17 +80,17 @@
...
-    <Style Selector="^:maximized /template/ Path#RestoreButtonPath">
+    <Style Selector="^:maximized /template/ Path#PART_RestoreButtonPath">
       <Setter Property="Data" Value="M2048 410h-410v-410h-1638v1638h410v410h1638v-1638zM1434 1434h-1229v-1229h1229v1229zM1843 1843h-1229v-205h1024v-1024h205v1229z" />
     </Style>
     <Style Selector="^:fullscreen /template/ Path#PART_FullScreenButtonPath">
       <Setter Property="IsVisible" Value="True" />
       <Setter Property="Data" Value="M205 1024h819v-819h-205v469l-674 -674l-145 145l674 674h-469v205zM1374 1229h469v-205h-819v819h205v-469l674 674l145 -145z" />
     </Style>
-    <Style Selector="^:fullscreen /template/ Panel#PART_RestoreButton">
+    <Style Selector="^:fullscreen /template/ Button#PART_RestoreButton">
```
Comment (robloo) [inline src/Avalonia.Themes.Fluent/Controls/CaptionButtons.xaml:90]: This and one below are correct fixes

File: src/Avalonia.Themes.Fluent/Controls/CaptionButtons.xaml (hunk 1 above)
Comment (robloo) [reply src/Avalonia.Themes.Fluent/Controls/CaptionButtons.xaml]: Thanks! Everything looks good.

**Comment by robloo** [comment]
Yea, I checked history. It's been broken a while. The template parts look correct now and match with code-behind:

https://github.com/AvaloniaUI/Avalonia/blob/4f6ef463693033cdeea206a4235a5cc847fb45e3/src/Avalonia.Controls/Chrome/CaptionButtons.cs#L11-L14

and

https://github.com/AvaloniaUI/Avalonia/blob/4f6ef463693033cdeea206a4235a5cc847fb45e3/src/Avalonia.Controls/Chrome/CaptionButtons.cs#L91-L94

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `11.0.999-cibuild0024882-beta`. (feed url: https://nuget.avaloniaui.net/repository/avalonia-all/index.json) [PRBUILDID]

**Review by maxkatz6 — APPROVED** [review]

**Comment by maxkatz6** [comment]
It seems to be broken in this PR, so it is isolated with 11.0
https://github.com/AvaloniaUI/Avalonia/commit/23a6517a6b064cecda957be0eeb768074bf76823

**Pull request merged**